packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -135,15 +153,19 @@
           ]
         },
         "utm_content": {
+          "description": "Per Google Analytics, this parameter is used to differentiate similar content, or links within the same ad. This parameter can be used to track the name of an A-B test (i.e., if a particular call-to-action is resulting in more traffic).",
```

Comment:
nit. Are these parameters only relevant for Google Analytics, or is that where the definition is from? If it's the later, could you add a note at the end saying something like "as defined by Google Analytics"?

Reply:
Here's an [explanation](https://mozilla-hub.atlassian.net/browse/FXA-6939?focusedCommentId=655412) of UTM parameters that Sean Rose shared a couple of weeks ago that may be helpful context here.